routes/transactionsRoute.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express')
+const router = express.Router()
+const transactionsController = require('../controllers/transactionsController.js')
+
+/**
+ * @swagger
+ * /transactions/fetchLatestTransactions:
```

Comment:
Please generate API schema for new route. Check scripts in package.json for the same.

Reply:
Generated